examples/grpc-bridge/README.md

```diff
@@ -1,2 +1,127 @@
 To learn about this sandbox and for instructions on how to run it please head over
 to the [envoy docs](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/grpc_bridge)
+
+# gRPC HTTP/1.1 to HTTP/2 bridge
+
+This is an example of a key-value store where a client CLI, written in Python, updates a remote store, written in Go, using the stubs generated for both languages. 
+ 
+Running clients that uses gRPC Stubs and sends messages through a proxy
+that upgrades the HTTP requests from http/1.1 to http/2. This is a more detailed
+implementation of the envoy documentation at https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/grpc_bridge
+
+* Client: talks in python and sends HTTP/1.1 requests (gRPC stubs)
+  * Client-Proxy: Envoy setup that acts as an egress and converts the HTTP/1.1 call to HTTP/2.
+* Server: talks in golang and receives HTTP/2 requests (gRPC stubs)
+  * Server-Proxy: Envoy setup that acts as an ingress and receives the HTTP/2 calls
+
+`[client](http/1.1) -> [client-egress-proxy](http/2) -> [server-ingress-proxy](http/2) -> [server]`
+
+# Running in 3 Steps
+
+* Generate Stubs: both the `client` and `server` stubs in `python` and `go` respectively to be used by each server.
+* Start Both Client and Server Servers and Proxies: `
+* Use the Client CLI to make calls to the kv server.
+
+## Generate Stubs
+
+* Uses the `protos` dir and generates the stubs for both `client` and `server`
+* Inspect the file `docker-compose-protos.yaml` with the gRPC protoc commands to generate the stubs.
```

Comment:
Would it be possible to have the docker files construct the stubs during their build? It seems like that would simplify  the setup here

Reply:
@snowp I tried a couple of approaches, but that would impose duplicating the protos... I wanted to show that gRPC is a 2-step process when working with client/server stubs... Because of the Docker Build context, we can't refer to directories outside the dir.